package.json

```diff
@@ -39,7 +39,7 @@
     "@material-ui/styles": "^4.11.4",
     "@reach/auto-id": "^0.16.0",
     "@saleor/macaw-ui": "npm:@saleor/macaw-ui@0.7.4",
-    "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@1.0.0-pre.18",
+    "@saleor/macaw-ui-next": "npm:@saleor/macaw-ui@1.0.0-pre.21",
```

Comment:
Is there a reason this PR bumps Macaw in this PR? I think its safer in general to avoid mixing too many things in single commit

Reply:
Good idea - I've backported this change in https://github.com/saleor/saleor-dashboard/pull/4643. We need version 21 as it fixes colors for warning banner while uninstalling apps.